LayoutTests/fast/frames/resources/self-referential-iframe.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style>
+        body, html {
+            height: 100%;
+            margin: 0;
+        }
+        .container {
+            height: 100%;
+        }
+        iframe {
+            display: block;
+            border: 0.5px solid gray;
+            width: 99.99%;
+            height: 99.99%;
+        }
+    </style>
+    <script>
+        window.addEventListener('load', () => {
+            let iframe = document.getElementsByTagName('iframe')[0];
+            let number = Math.random();
```

Comment:
There’s no strong guarantee we won’t have collisions with `random`. Is there another way to do this?

Reply:
A better way to do this would be to increment the number from this frame for the next frame.  We could then also use this to do something like verify that load fails exactly where we want it to, then postMessage a success message back up the frame tree.